tools/perspective-test/playwright.config.ts

```diff
@@ -226,7 +226,7 @@ export default defineConfig({
     snapshotPathTemplate:
         "dist/snapshots/{projectName}/{testFilePath}/{arg}{ext}",
     webServer: {
-        command: "node --loader ts-node/esm src/js/start_test_server.ts",
+        command: "tsx src/js/start_test_server.ts",
```

Comment:
When I was adjusting the playwright config, `node --loader` was driving me a little crazy with obscure errors. tsx was already in the dependency list and it worked a lot better for me.